src/main/java/io/stargate/sgv2/jsonapi/service/resolver/CreateIndexCommandResolver.java

```diff
@@ -86,6 +87,7 @@ public Operation resolveTableCommand(
     return new GenericOperation<>(
         new OperationAttemptContainer<>(attempt),
         pageBuilder,
-        new CreateIndexExceptionHandler(apiIndex.indexName()));
+        DefaultDriverExceptionHandler.Factory.withIdentifier(
+            CreateIndexExceptionHandler::new, apiIndex.indexName()));
```

Comment:
Whoa. This took time to unpack. Functional style is compact but not necessarily easy to digest.
(not sure there's any way to make it more readable, just an observation).

Reply:
understood, it makes it simple to use... but not simple for implementation. 

(I got the idea from the Pragmatic Java book talking about  method references for a ctor :) )